lib/manager/jsonnet-bundler/artifacts.spec.ts

```diff
@@ -0,0 +1,200 @@
+import { exec as _exec } from 'child_process';
+import { join } from 'upath';
+import { envMock, mockExecAll } from '../../../test/exec-util';
+import { fs, git, mocked } from '../../../test/util';
+import { setGlobalConfig } from '../../config/global';
+import { RepoGlobalConfig } from '../../config/types';
+import * as _env from '../../util/exec/env';
+import { StatusResult } from '../../util/git';
+import { UpdateArtifactsConfig } from '../types';
+import { updateArtifacts } from '.';
+
+const exec: jest.Mock<typeof _exec> = _exec as any;
+const env = mocked(_env);
+jest.mock('child_process');
+jest.mock('../../util/exec/env');
+jest.mock('../../../lib/util/fs');
+jest.mock('../../../lib/util/git');
```

Comment:
Some import are wrong here
```suggestion
jest.mock('../../util/exec/env');
jest.mock('../../util/fs');
jest.mock('../../util/git');
```